arrays.js

```diff
@@ -68,11 +93,34 @@ function sortStudentsAlphabetically(students) {
    - console.log(odds); // Output: [1, 3, 5, 7, 9]
    - console.log(evens); // Output: [2, 4, 6, 8, 10]
 **************************************************************/
+var array_odd=[];
```

Comment:
Well done, we mentioned more than once not using (var) because it will result in unexpected behavior sometimes. Please stick to the instructors' instructions